### PR TITLE
fix and polish for installation [+]

### PR DESCRIPTION
**Pull request by fenekku**
- fix packaging
- rename and run tests
- add tasks
- release as v2023.1.2

### CODE REVIEW

**Review by Meowcenary — APPROVED** [review]
Looks good to me! :shipit:

File: README.md [hunk 1]
```diff
@@ -87,10 +89,22 @@ Run tests:
...
+# or if using pipenv
+pipenv run inv check-manifest
+```
+
+Clean out artefacts:
+
+```bash
+(venv) inv clean
+# or if using pipenv
+pipenv run inv clean
```
Comment (sharpattack) [inline README.md:109]: How often, or when, should artifacts be cleaned?

**Review by sharpattack — APPROVED** [review]
Looks good to me, thanks.

File: README.md (hunk 1 above)
Comment (fenekku) [reply README.md:109]: When building locally and mucking around with editable and non-editable installs, cleaning is a good habit. The artifacts cause a lot of weird behavior, that sapped good chunks of time for me this week. That being said, building is typically done via the GitHub Actions and local development (running tests) via directives mentioned doesn't typically need it. So I wouldn't worry about it too much on a day-to-day development. Only when doing more involved package debugging is this really useful.

**Pull request merged**